### PR TITLE
Enable periodic builds for CUDA 11.7

### PR DESCRIPTION
**Pull request by ptrblck**
CC @atalman

### CODE REVIEW

File: .circleci/cimodel/data/dimensions.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "102",
     "113",
     "116",
+    "117",
```
Comment (janeyx99) [inline .circleci/cimodel/data/dimensions.py:7]: This file doesn't seem like it's used anymore.

**Review by atalman — APPROVED** [review]

File: .github/scripts/install_nvidia_utils_linux.sh [hunk 2]
```diff
@@ -3,7 +3,7 @@
 set -eou pipefail
 
 DISTRIBUTION=$(. /etc/os-release;echo $ID$VERSION_ID) \
-DRIVER_FN="NVIDIA-Linux-x86_64-510.60.02.run"
+DRIVER_FN="NVIDIA-Linux-x86_64-515.57.run"
```
Comment (malfet) [inline .github/scripts/install_nvidia_utils_linux.sh:6]: Why are we updating driver here? Aren't forward compatibility guarantee for CUDA-11.x applies for 11.7?

Comment (malfet) [inline .github/scripts/install_nvidia_utils_linux.sh:6]:
```suggestion
sudo modprobe backlight
DRIVER_FN="NVIDIA-Linux-x86_64-515.57.run"
```

**Pull request closed without merging**